react/features/app/actions.js

```diff
@@ -14,11 +14,23 @@ import {
 } from './actionTypes';
 import {
     _getRoomAndDomainFromUrlString,
-    _getRouteToRender
+    _getRouteToRender,
+    init
 } from './functions';
 import './reducer';
 
 /**
+ * Temporary solution. Should dispatch actions related to
+ * initial settings of the app like setting log levels,
+ * reading the config parameters from query string etc.
+ *
+ * @returns {Function}
+ */
+export function appInit() {
```

Comment:
I'm fully agree with you. It's a temporary solution because app initialization is not fully integrated in new app. And in one of further PRs based on this one I refactored this action to just function.